website/docs/configuration/flags.md

```diff
@@ -9,6 +9,16 @@ You can always see the complete list by using `--help` flag.
 To make user experience cloud native, every flag has its environment variable equivalent.
 There is no configuration file.
 
+However, you can use a bash script to configure and run FerretDB. Here's an example:
+
+```bash
+#!/bin/sh
+export FERRETDB_POSTGRESQL_URL=xxx
+exec ferretdb
+```
+This approach avoids long command lines and improves clarity.
+The use of `exec` allows the FerretDB process to correctly receive and handle termination signals (like SIGTERM). This is particularly important when running FerretDB as PID 1 in containers, ensuring a clean and graceful shutdown.
```

Comment:
While the part about containers is correct, why would one add a script to the container already configured by environment variables?

A more realistic example would be someone starting the FerretDB binary manually on the host, ensuring that command-line flags are not lost in the shell history and that the environment is not reset in a new terminal window.